test/libqlever/QleverTest.cpp

```diff
@@ -12,6 +12,118 @@
 using namespace qlever;
 using namespace testing;
 
+// _____________________________________________________________________________
+TEST(LibQlever, buildIndexAndRunQuery) {
+  std::string filename = "testIndexForLibQlever.ttl";
+  {
+    auto ofs = ad_utility::makeOfstream(filename);
+    ofs << "<s> <p> <o>. <s2> <p> \"kartoffel und salat\".";
+  }
+
+  IndexBuilderConfig c;
+  c.inputFiles_.emplace_back(filename, Filetype::Turtle);
+  c.baseName_ = "testIndexForLibQlever";
+
+  // Test the activation of the memory limit
+  c.memoryLimit_ = ad_utility::MemorySize::bytes(0);
+  EXPECT_ANY_THROW(Qlever::buildIndex(c));
```

Comment:
Please either add a comment why any_throw is the correct assertion here, or expect a fixed exception type. Otherwise this function might throw for a wrong reason without this test failing. Same for the other ANY_THROW in this file)